2.1/alpine/Dockerfile

```diff
@@ -78,6 +78,7 @@ RUN set -ex \
 		scanelf --needed --nobanner --recursive /usr/local \
 			| awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
```

Comment:
Thanks for finding the bug, but I would like to find a general solution to apply it to images other than ruby. Many official images that build from source on alpine use these same `scanelf` lines and might be have a similar problem.

:thinking: :cry: